devito/operator/operator.py

```diff
@@ -648,7 +648,8 @@ def _postprocess_arguments(self, args, **kwargs):
                 subfuncs = (args[getattr(p, s).name] for s in p._sub_functions)
                 p._arg_apply(args[p.name], *subfuncs, alias=kwargs.get(p.name))
             except AttributeError:
-                p._arg_apply(args[p.name], alias=kwargs.get(p.name))
+                if getattr(p, 'parent', None) not in self.parameters:
```

Comment:
`if isinstance ( .. SubFunction)`

Avoid `getattr` except in peculiar situations (like list comprehensions and such )